<launch>

    <!-- vrpn Optitrack -->
    <arg name="server" default="128.178.145.104"/>

    <param name="robot_description"
    textfile="$(find optitrack_object_pose)/urdf/panda_mass.urdf" />
    
    <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" >
        <rosparam subst_value="true">
        server: $(arg server)
        port: 3883
        frame_id: world
        broadcast_tf: true
        </rosparam>
    </node>
    <node name="Initialize_pose_franka" pkg="optitrack_object_pose" type="Initialize_pose_franka" output="screen"/> 

</launch>
